site/versions/master/docs/windows.md

```diff
@@ -3,42 +3,27 @@ layout: documentation
 title: Windows
 ---
 
-Building Bazel on Windows
-=========================
-
 Windows support is highly experimental. Known issues are [marked with
 label "Windows"](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+is%3Aopen+label%3A%22category%3A+multi-platform+%3E+windows%22)
 on github issues.
 
-We currently support only 64 bit Windows 7 or higher and we compile Bazel as a msys2 binary.
 
-To bootstrap Bazel on Windows, you will need:
+Installing Bazel on Windows
+===========================
 
-*    Java JDK 8 or later
-*    [Visual Studio](https://www.visualstudio.com/) (Community Edition is okay)
-*    [msys2](https://msys2.github.io/) (need to be installed at
-     ``C:\tools\msys64\``).
-*    Several msys2 packages. Use the ``pacman`` command to install them:
-     ``pacman -S gcc git curl zip unzip zlib-devel``
-
-To build Bazel:
-
-*    Open the msys2 shell.
-*    Clone the Bazel git repository as normal.
-*    Set the environment variables:
+You can install using the [chocolatey](https://chocolatey.org) package manager:
```

Comment:
For now, keep building from source the preferred method of installation in this do.
Mention that unofficial chocolatey package is avialable.